apps/site/lib/site_web/views/page_view.ex

```diff
@@ -10,6 +10,143 @@ defmodule SiteWeb.PageView do
 
   use SiteWeb, :view
 
+  @spec sort_string_number(String.t(), String.t()) :: boolean
+  defp sort_string_number(a, b) do
```

Comment:
this could use guard clauses to remove the need for a case statement

```elixir
defp sort_string_number(a, b) when is_integer(a) and is_integer(b) do
  a <= b
end
defp sort_string_number(a, b), do: a <= b
```

Reply:
Ah, yeah you're right, I missed that `Integer.parse/2` returns a tuple with the remainder. It does happen to sort properly, but I'll adjust the code to make it more obvious, thanks!